### ISSUE TITLE
Usability Issues: UI based on colors

### ISSUE THREAD

**Issue by anibalsanchez** [joomla-extensions/jedchecker#154]
Now, that we are polishing the new UI, the UI is currently based on colors (I'm color blind). There are concepts that are only expressed with colors (green, yellow, or red).

It would be better if the notices are always marked with labels "ERROR:", "WARN:", "INFO:" and with Icons for better signaling.

![alerts](https://user-images.githubusercontent.com/385780/132088644-1506c4df-8109-447b-a6d1-dba75324fa8b.png)

**Comment by Llewellynvdm** [comment]
Yes, I think this will be a good choice.

**Comment by dryabov** [comment]
The problem is that Joomla4 is shipped with FontAwesome 5, so that we can use its icons easily, but for Joomla3 we need to either include FontAwesome into the package, or use another set of icons for both Joomla 3 and 4 (in the latter case, we could display icons as data-uri-encoded svg in the `backgound-image` CSS property).

And know "standard" icons for Error/Warning/Info level, but I'm not sure which one to use for Deprecated level. Do you have ideas?

Maybe the easiest solution is to just add a "badge" text on top of existing blocks, e.g.
![v1](https://user-images.githubusercontent.com/378044/132100195-ba7d24ba-5bb7-4701-808f-f49d55f74913.png)
or
![v2](https://user-images.githubusercontent.com/378044/132100200-a441bdac-56d5-43ce-ab02-cea80efc90ef.png)
that were done with the following CSS snippet:
```css
#jedchecker .alert-warning::before {
    content: "WARNING";
    font-size: .75rem;
    line-height: 1.2;
    padding: 0 .2rem;
    border-radius: .2rem;
    background-color: #664808;
    color: #fff0d0;
    position: absolute;

    /* Ver.1 */
    /*
    top: .1rem;
    left: .1rem;
    opacity: 0.75;
    */

    /* Ver.2 */
    top: -0.5rem;
    left: .5rem;
}
```

**Comment by Llewellynvdm** [comment]
The second option will be okay, should just be larger.
